src/api/kube.ts

```diff
@@ -1400,7 +1498,7 @@ export class KubeHelper {
     return crdv ? crdv.name : 'v1'
   }
 
-  async deleteCrd(name: string): Promise<void> {
+  async deleteCrdV1Beta1(name: string): Promise<void> {
```

Comment:
What if a resource with `v1alpha1` or `v1alpha2` appear? What if it then turned into `v1beta2`?
According to the current strategy, we'll have to add several methods per version.
To me it is better to have version as second parameter (maybe default to `v1`)

Reply:
Agree. But let's keep as is for now.

Reply:
ok